app/values/iiif_path.rb

```diff
@@ -9,6 +9,10 @@ def to_s
     "#{iiif_url}/#{escaped_path}"
   end
 
+  def thumbnail
+    "#{self}/full/100,/0/default.jpg"
```

Comment:
Yeah. If we have UI constraints that need the image to be at or smaller that 200 x 150, that's the way to go. For portrait images, thanks to some cache magic, Loris is going to serve the same file for `w,` and `!w,h` anyway.